Ktisis/Interface/Windows/Workspace.cs

```diff
@@ -152,7 +157,6 @@ public class Workspace {
 
 			// Use the same functions found in SkeletonEditor.Draw()
 			var delta = KtisisGui.SkeletonEditor.BoneMod.GetDelta();
-			selectedBone.Transform.Rotation *= delta.Rotation;
```

Comment:
Is this related to the animation pausing changes or a separate change?

Reply:
This one was a mistake.